code/Example/Drasil/Sections/TableOfUnits.hs

```diff
@@ -10,7 +10,7 @@ import Data.Drasil.Concepts.Documentation
 -- | Table of units section builder. Takes a list of units and an introduction
 table_of_units :: IsUnit s => [s] -> Contents -> Section
 table_of_units u intro = Section (S "Table of Units") [Con intro, Con (unit_table u)] 
-  "ToU" "TblOfUnits"
+  "ToU" (shortname' "TblOfUnits")
```

Comment:
@JacquesCarette @szymczdm We do want the ability to pass in something different from the title and the `reference address`, correct? Currently, "TblOfUnits" isn't seen in the html or latex files...